blocks/newsletter-subscribe/newsletter-subscribe.css

```diff
@@ -0,0 +1,371 @@
+  background: var(--circle-svg), #1f2022;
+  background-size: 22px 22px;
+  background-repeat: no-repeat;
+  background-position: 26px 4px;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper h1,
+.newsletter-subscribe.block .subscribe-form .title-wrapper h2,
+.newsletter-subscribe.block .subscribe-form .title-wrapper h3 {
+  font-family: var(--font-tungsten);
+  font-size: 67px;
+  font-weight: 600;
+  line-height: 66px;
+  letter-spacing: 0.05em;
+  text-transform: uppercase;
+  color: black;
+  margin: 0 0 10px;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="description"] p {
+  font-family: "Times New Roman", Times, serif;
+  text-align: center;
+  max-width: 500px;
+  font-style: normal;
+  font-weight: 325;
+  font-size: 18px;
+  margin: 0;
+  color: #4e515e;
+  line-height: 30px;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="tos-top"] {
+  color: #1f2022;
+  text-align: center;
+  font-family: "Times New Roman", Times, serif;
+  font-size: 10px;
+  line-height: 13px;
+  letter-spacing: 0;
+  margin-bottom: 20px;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="tos-top"] a {
+  font-weight: bold;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="tos-top"] p {
+  margin: 0;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="description"] p:first-child {
+  line-height: 135%;
+  color: #151517;
+}
+
+.newsletter-subscribe.block .subscribe-form .title-wrapper [slot="description"] p:last-child {
+  margin-bottom: 30px;
+}
+
+.newsletter-subscribe.block .subscribe-form .controls-wrapper {
+  width: 100%;
+  max-width: 600px;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+}
+
+.newsletter-subscribe.block .subscribe-form .controls-wrapper .email-input {
+  height: 70px;
+  width: 100%;
+  border: 1px solid #adb4b7;
+  background-color: #fff;
+  padding-left: 7px;
+  box-sizing: border-box;
+  text-indent: 5px;
+  box-shadow: inset 0 1px 3px rgba(0 0 0 / 5%);
+  color: #4e515e;
+  outline: none;
+  font-size: 18px;
+  line-height: 30px;
+  font-family: "Times New Roman", Times, serif;
+}
+
+.newsletter-subscribe.block .subscribe-form .controls-wrapper .email-input:focus {
+  border: 1px solid #87b9ea;
+  outline: none;
+  box-shadow: 0 0 5px 0 #1d87ff;
+}
+
+.newsletter-subscribe.block .subscribe-form .controls-wrapper .email-input::placeholder {
+  color: #adb4b7;
+}
+
+.newsletter-subscribe.block .subscribe-form .controls-wrapper .email-input.invalid {
+  border-style: solid;
+  border-width: 1px;
+  border-color: #e76468;
+  box-shadow: none;
+}
+
+.newsletter-subscribe.block .subscribe-form .controls-wrapper .email-input.valid {
+  background-image: url("https://cdns1.gigya.com/gs/i//screenSet/checkmarkValid.png");
```

Comment:
Instead of relying on cdns1.gigya.com, you could transform the image into a base64 string so at least it will always work independent of cdns1.gigya.com. 
Or you could publish it so it uses frnaklin but maybe that's a bit of a hassle for such a small image. Up to you.

Reply:
now uses base64